packages/runtime-tools-management-console-webapp/src/tasks/components/ImpersonationPageSection.tsx

```diff
@@ -153,38 +155,41 @@ export const ImpersonationPageSection: React.FC<{}> = () => {
               />
             }
           >
-            <FormGroup label={"User"}>
+            <FormGroup label={"User"} style={{ maxWidth: "500px" }}>
               <TextInput
                 className={
                   username && username === impersonationUsername
-                    ? "pf-c-form-control pf-m-success"
-                    : "pf-c-form-control pf-m-expanded"
+                    ? "pf-v5-c-form-control pf-v5-m-success"
+                    : "pf-v5-c-form-control pf-v5-m-expanded"
```

Comment:
It seems pf-m-success and pf-m-expanded have no `-v5` prefix in v5
```suggestion
                    ? "pf-v5-c-form-control pf-m-success"
                    : "pf-v5-c-form-control pf-m-expanded"
```